src/components/CustomModal,/CustomModal.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text
+} from '@chakra-ui/react';
+import i18next from 'i18next';
+
+export const CustomModal = ({ isOpen, onClose, onSubmit, title, text }) => {
+  return (
+    <Modal
+      closeOnOverlayClick={false}
+      isOpen={isOpen}
+      onClose={onClose}
+      isCentered
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>{title}</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody>
+          <Text fontWeight="bold" mb="1rem">
+            {text}
+          </Text>
+        </ModalBody>
+
+        <ModalFooter>
+          <Button colorScheme="red" onClick={onSubmit}>
+            {i18next.t('Yes')}
```

Comment:
make this a label in the i18n_en.json

Reply:
it will work only because i18n couldn't find the key so it returns the string you provided, that is not how you use i18n